### ISSUE TITLE
Not able to transfer or approve a transaction

### ISSUE THREAD

**Issue by chsivateja** [ontio/cyano-wallet#45]
I tried using cyano wallet for ONT. Transferred from exchange to wallet. But not able to transfer then back to exchange. Neither can I swap ONT to ONTD on wing website.

**Comment by dannjager** [comment]
same problem for me with ledger. windows security keeps popping up.

**Comment by koholintian** [comment]
I have this problem too. It doesn't recognize my ledger wallet when I try to unwrap pONT for ONT on flamingo.finance.

**Comment by jshendwg** [comment]
Yes my ONT is stuck as well after polynetwork unwrap, please provide a solution for fixing the ledger controlled wallet, it'll be difficult to get the private key out of ledger to import in the extension wallet manually.
